docs/guides/guides/screenshots-and-videos.mdx

```diff
@@ -43,21 +43,28 @@ to clear your screenshots folder before a run, you can set
 
 ## Videos
 
-Cypress records a video for each spec file when running tests during
-`cypress run`. Videos are _not_ automatically recorded during `cypress open`.
+If enabled, Cypress records a video for each spec file when running tests during
+`cypress run`. Videos are _not_ recorded during `cypress open`.
 
-Video recording can be turned off entirely by setting
-[`video`](/guides/references/configuration#Videos) to `false` from within your
+Video recording can be turned on by setting
```

Comment:
```suggestion
Video recording is disabled by default, but can be turned on by setting
```

Reply:
I think this should be the first sentence to flow these a bit better. its disabled > heres how you turn it on > when it's on, here is the behavior.

Reply:
@emilyrohrbough how is 1b5358019?